//
// PROJECT:         Aspia Remote Desktop
// FILE:            proto/power_session.proto
// LICENSE:         Mozilla Public License Version 2.0
// PROGRAMMERS:     Dmitry Chapyshev (dmitry@aspia.ru)
//

syntax = "proto3";

option optimize_for = LITE_RUNTIME;

import "power_session_message.proto";

package aspia.proto.power;

message ClientToHost
{
    PowerEvent power_event = 1;
}
